internal/ingress/controller/parser/parser.go

```diff
@@ -268,7 +269,9 @@ func getCerts(log logrus.FieldLogger, s store.Storer, secretsToSNIs map[string][
 				CreationTimestamp: secret.CreationTimestamp,
 			}
 		} else {
-			if kongCert.CreationTimestamp.After(secret.CreationTimestamp.Time) {
+			if kongCert.CreationTimestamp.After(secret.CreationTimestamp.Time) ||
+				(kongCert.CreationTimestamp.Equal(&secret.CreationTimestamp) &&
+					*kongCert.cert.ID > string(secret.UID)) {
```

Comment:
Question:
What is the logic behind this comparison?
Is there a reason why you have not opted for `!=`?

(blocker for merge)

Reply:
![image](https://user-images.githubusercontent.com/25172225/95419504-590bde00-096c-11eb-9ea3-2cfa4963b1ec.png)
![image](https://user-images.githubusercontent.com/25172225/95419544-69bc5400-096c-11eb-849c-1829314eb75a.png)
As shown in the figure above, when the certificate name and create_time of different namespace When time is consistent, there are problems
![image](https://user-images.githubusercontent.com/25172225/95420073-6d040f80-096d-11eb-9fbf-b86e15575a22.png)
Because secretsToSNIs is a map, it is unordered